jwst/regtest/test_miri_caldector1.py

```diff
@@ -0,0 +1,93 @@
+from glob import glob
+import os
+
+import pytest
+from astropy.io.fits.diff import FITSDiff
+from jwst.pipeline import Detector1Pipeline
+from jwst.pipeline.collect_pipeline_cfgs import collect_pipeline_cfgs
+from jwst.stpipe import Step
+
+
+@pytest.mark.bigdata
+def test_miri_caldetector1_rate(_jail, rtdata, fitsdiff_default_kwargs):
+    rtdata.get_data("miri/image/jw00001001001_01101_00001_MIRIMAGE_uncal.fits")
+
+    collect_pipeline_cfgs("config")
+    args = ["config/calwebb_detector1.cfg", rtdata.input]
+    Step.from_cmdline(args)
+    rtdata.output = "jw00001001001_01101_00001_MIRIMAGE_rate.fits"
+
+    rtdata.get_truth("truth/test_miri_caldetector1/jw00001001001_01101_00001_MIRIMAGE_rate.fits")
+    assert rtdata.output != rtdata.truth
```

Comment:
I don't think you need this. It's just comparing the output and truth file names, which should be the same except for the path.